aries_cloudagent/protocols/issue_credential/v1_0/manager.py

```diff
@@ -752,27 +752,15 @@ def context(self) -> InjectionContext:
 
         registry_records = await IssuerRevRegRecord.query_by_pending(self.context)
         for registry_record in registry_records:
-            net_delta = {}
-            for cr_id in registry_record.pending_pub:
-                delta = json.loads(
-                    await issuer.revoke_credential(
-                        registry_record.revoc_reg_id,
-                        registry_record.tails_local_path,
-                        cr_id,
-                    )
-                )
-                if delta:
-                    net_delta = (
-                        await issuer.merge_revocation_registry_deltas(net_delta, delta)
-                        if net_delta
-                        else delta
-                    )
-
-            registry_record.revoc_reg_entry = net_delta
+            revoke_idxs = list(registry_record.pending_pub)
+            delta_json = await issuer.revoke_credentials(
+                registry_record.revoc_reg_id,
+                registry_record.tails_local_path,
+                revoke_idxs,
+            )
+            registry_record.revoc_reg_entry = json.loads(delta_json)
             await registry_record.publish_registry_entry(self.context)
-            result[registry_record.revoc_reg_id] = [
-                cr_id for cr_id in registry_record.pending_pub
-            ]
+            result[registry_record.revoc_reg_id] = revoke_idxs
```

Comment:
Doesn't this clobber any prior pending revocations? They won't make it into the delta, and they won't be pending anymore.

Reply:
I'm not sure I understand. It's doing the same thing as before, creating a combined delta from all the pending indexes, just doing that inside the IndyIssuer class instead. The credx issuer has an optimized path that creates a combined delta all at once.

Reply:
Oh - I missed a line. It's fine.